src/pages/read/[linkId].tsx

```diff
@@ -2,14 +2,17 @@ import React from "react";
 import ReaderLayout from "~/src/components/ReaderLayout/ReaderLayout";
 import { readFetcher } from "~/src/lib/api/SearchApi";
 import { LinkResult } from "~/src/types/LinkQuery";
+
 export async function getServerSideProps(context: any) {
+  console.log("linkId", context.query);
```

Comment:
somehow `context.query.linkId` is getting the link (http://localhost:3000/read/6551603) as well as pdf.min.js.  This is throwing an error `Error: cannot find work with linkId pdf.worker.min.js` 

Something needs to be set up such that it looks in the right place?